app/controllers/todo_items_controller.rb

```diff
@@ -58,6 +58,22 @@ def update
     end
   end
 
+  def destroy
+    todo_item = TodoItem.find(params[:id])
```

Comment:
What are your thoughts about handling the 404/ActiveRecord::RecordNotFound exception? What are the reasons to consider it, and what is the standard behavior for that?